app/assets/v2/js/metamask-approval.js

```diff
@@ -0,0 +1,25 @@
+async function metamaskApproval() {
+  if (window.ethereum) {
+    window.web3 = new Web3(ethereum);
+    is_metamask_approved = await window.ethereum._metamask.isApproved();
+    is_metamask_unlocked = await window.ethereum._metamask.isUnlocked();
+
+    try {
+      if (is_metamask_unlocked && is_metamask_approved) {
+        await ethereum.enable();
+      }
+    } catch (error) {
+
+    }
+  }
+}
+
+window.addEventListener('load', metamaskApproval);
+
+async function approve_metamask() {
+  try {
+    await ethereum.enable();
+  } catch (error) {
+
```

Comment:
console.log the error here ? / throw an alert stating user disapproved ?  something like 
`_alert("Looks like you've disabled metamask. Enable it to allow Gitcoin to interact with metamask ", "warning");`

Reply:
cc @mbeacom

Reply:
Is console.log allowed? I guess linting will give an error right?

Reply:
Yeah it would ! that's why I suggested the alert :P we can add that in later if needed unless you wanna add it in now

Reply:
What does `_alert()` do?

Reply:
it shows a message to the user at the top of the page

Reply:
It does not seem to work for me. I have added it already into the  PR